src/hooks/operators.ts

```diff
@@ -551,3 +534,60 @@ const mapOperatorOrder = (orderBy: string | undefined): Operator_OrderBy => {
             return Operator_OrderBy.Id
     }
 }
+
+/**
+ * Returns a callback that takes the user through the process of creating
+ * an operator or updating an existing operator.
+ */
+export function useTouchOperatorCallback() {
```

Comment:
I'm not sure about the name of this callback 🤔 I'd say `touching` makes me think of updating only timestamps or so.

Reply:
Yeah, you may be right, I need a better name.

Reply:
`useSaveOperator`, `usePersistOperator`, any other ideas?

Reply:
I'd vote for good old `useSaveOperator` 😄